src/driver/compile.c

```diff
@@ -396,7 +396,8 @@ static int yf_compile_files(struct yf_args * args, struct yf_compilation_data *
         fdata->file_name = yf_strdup(fname);
         fdata->parse_anew = 1;
         /* TODO - more data */
-        yfh_set(&data.files, fdata->file_name, fdata);
+        if (yfh_set(&data.files, fdata->file_name, fdata) != YF_OK)
+            abort();
```

Comment:
Could we print some error message here indicating what's happening? Or does `abort` do something similar?

Reply:
Abort lets us break if we're debugging. For a regular user, the error would be meaningless since there's not anything that they can do anything about it.